### PR TITLE
feat: add basic support for clients

### PR DESCRIPTION
**Pull request by sascha-andres**
This PR adds basic support for clients. Basic in the way that you can create a client and list all clients. Nothing more

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -544,6 +544,48 @@ func (session *Session) DeleteTag(tag Tag) ([]byte, error) {
 	return session.delete(TogglAPI, path)
 }
 
+// GetClients returns a list of clients for the current account
+func (session *Session) GetClients() ([]Client, error) {
```
Comment (jason0x43) [inline main.go]: How would you feel about `(clients []Client, err error)` as the return type, and then using the `clients` variable in the `return` statements in the function?

File: main.go [hunk 2]
```diff
@@ -857,3 +899,4 @@ func EnableLog() {
 	dlog.SetFlags(logFlags)
 	dlog.SetOutput(os.Stderr)
 }
+
```
Comment (jason0x43) [inline main.go]: This extra line isn't needed.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (jason0x43) [inline .gitignore]: I'd prefer not to add this .gitignore since it's not generally applicable.

**Review by jason0x43 — CHANGES_REQUESTED** [review]
Overall this seems good. I'd prefer to leave out the .gitignore updates, though, and I have a couple small code suggestions.

File: main.go (hunk 1 above)
Comment (sascha-andres) [reply main.go]: actually makes the code more readable - changed accordingly

File: .gitignore (hunk 3 above)
Comment (sascha-andres) [reply .gitignore]: removed

File: main.go (hunk 2 above)
Comment (sascha-andres) [reply main.go]: should be removed - probably added by formatting the code. Have to dig into my settings there

**Comment by sascha-andres** [comment]
@jason0x43 Made requested changes

**Review by jason0x43 — APPROVED** [review]

**Pull request merged**